ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/HDFS/package/files/checkWebUI.py

```diff
@@ -67,19 +67,21 @@ def main():
   parser.add_option("-p", "--port", dest="port", help="Port of WEB UI to check it availability")
   parser.add_option("-s", "--https", dest="https", help="\"True\" if value of dfs.http.policy is \"HTTPS_ONLY\"")
   parser.add_option("-o", "--protocol", dest="protocol", help="Protocol to use when executing https request")
+  parser.add_option("-a", "--path", dest="path", help="Path of WEB UI")
 
   (options, args) = parser.parse_args()
   
   hosts = options.hosts.split(',')
   port = options.port
   https = options.https
   protocol = options.protocol
+  path = options.path
 
   for host in hosts:
-    httpCode = make_connection(host, port, https.lower() == "true", protocol)
+    httpCode = make_connection(host, port, https.lower() == "true", protocol, path)
 
     if httpCode != 200:
```

Comment:
Hi @smallyao , after I take a deep look at this bug, I found the error only occurs when Kerberos is disabled and HA is enabled, it reports an error when http response code is 200, but now it returns 302, which I think also should be healthy status.
Can we simply add one more condition here to make 302 passed too?

Reply:
Hi @kevinw66  It's also okay to make http response code 302 pass if just simply resolve this issue.  I will modify it.